gsl_simulation/templates/includes/_refuse_confirmation_modal.html

```diff
@@ -0,0 +1,41 @@
+<dialog id="refuse-confirmation-modal" class="fr-modal">
+    <div class="fr-container fr-container--fluid fr-container-md">
+        <div class="fr-grid-row fr-grid-row--center">
+            <div class="fr-col-12 fr-col-md-8 fr-col-lg-6">
+                <div class="fr-modal__body">
+                    <div class="fr-modal__header">
+                        <button aria-controls="modal"
+                                title="Fermer"
+                                type="button"
+                                id="button-5"
+                                class="fr-btn--close fr-btn"
+                                onClick="closeModal('refuse-confirmation-modal')">
+                            Fermer
+                        </button>
+                    </div>
+                    <div class="fr-modal__content">
+                        <h1 id="modal-title" class="fr-modal__title refuse-color">
+                            <span class="fr-icon-checkbox-fill fr-icon--lg" aria-hidden="true"></span> Refuser le financement du projet
+                        </h1>
+                        <p>
+                            Ce projet va être refusé dans toutes les simulations dans lesquelles il a été ajouté.
+                            Il n'apparaîtra pas dans la programmation officielle DETR {{ simulation_year }}.
```

Comment:
DETR est hardcodé : 

```suggestion
                            Il n'apparaîtra pas dans la programmation officielle {{ simulation_enveloppe.type }} {{ simulation_year }}.
```

(Ne pas commit comme ça, je ne sais pas ce qu'on a dans le contexte !)